Cargo.toml

```diff
@@ -16,7 +16,7 @@ categories = ["network-programming", "game-development"]
 wasm-bindgen = ["instant/wasm-bindgen", "ggrs/wasm-bindgen"]
 
 [dependencies]
-bevy = { version = "0.7", default-features = false }
+bevy = { version = "0.8.0", default-features = false, features = ["render", "bevy_asset","bevy_scene",]}
```

Comment:
Why do these need to be added to the dependencies? Can't they just be dev dependencies?

Reply:
Yes, should work fine as dev dependencies.